src/libsyntax/feature_gate.rs

```diff
@@ -2508,6 +2511,18 @@ pub fn check_crate(krate: &ast::Crate,
         parse_sess: sess,
         plugin_attributes,
     };
+
+    sess
+        .param_attr_spans
+        .borrow()
+        .iter()
+        .for_each(|span| gate_feature!(
+            &ctx,
+            param_attrs,
+            *span,
+            "attributes on formal function parameters are unstable"
```

Comment:
```suggestion
            "attributes on function parameters are unstable"
```
Our diagnostics are using the "parameter" vs "argument" distinction rather than "formal" in their wording.